src/nick.c

```diff
@@ -34,7 +34,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 User *find_account(const Nick *nptr)
 {
-    return nptr->account ?: find_user(nptr->nick);
+    return find_user(nptr->svid) ?: find_user(nptr->nick);
```

Comment:
nptr->account should be strictly equivalent to find_user(nptr->svid). In which situation does this no longer hold?

Reply:
Good question, I can't remember why `nptr->account` didn't work in my case, I'll test it a bit more.